tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "isolatedModules": true,
     "noEmit": true,
     "noImplicitThis": false,
+    "noImplicitAny": false,
```

Comment:
Id prefer to not add this, why was it added?

Reply:
Done